db/migrate/20171026114327_add_deleted_on_to_container_quota_and_items.rb

```diff
@@ -1,9 +1,34 @@
 class AddDeletedOnToContainerQuotaAndItems < ActiveRecord::Migration[5.0]
+  class ContainerQuota < ActiveRecord::Base; end
+  class ContainerQuotaItem < ActiveRecord::Base
+    belongs_to :container_quota
```

Comment:
Here, you have to specify the `:class_name => "AddDeletedOnToContainerQuotaAndItems::ContainerQuota"`